### PR TITLE
Renamed kav's board.json file and customDevice types

### PR DESCRIPTION
**Pull request by elral**
Rename board.json file for KAV's displays and rename accordingly the device types in the device.json file

### CODE REVIEW

**Comment by elral** [comment]
Fixed also a wrong MF type in generic I2C

File: Boards/kav_mega.board.json.inactive [hunk 1]
```diff
@@ -35,15 +35,15 @@
     "CanInstallFirmware": true,
     "CanResetBoard": true,
     "DelayAfterFirmwareUpdate": 0,
-    "FirmwareBaseName": "kav_efis_fcu_mega",
+    "FirmwareBaseName": "kav_mega",
```
Comment (DocMoebiuz) [inline Boards/kav_mega.board.json.inactive:38]: the firmware file has to be renamed too:
https://github.com/elral/MobiFlight-Connector/blob/renaming_KAV_customDevices/firmware/kav_efis_fcu_mega_2_4_4.hex

**Review by DocMoebiuz — CHANGES_REQUESTED** [review]
only the FW file name has to be changed. the rest is looking great!

**Review by DocMoebiuz — APPROVED** [review]
Dankeschön

**Pull request merged**